ui/components/app/advanced-gas-fee-popover/index.scss

```diff
@@ -0,0 +1,19 @@
+.advanced-gas-fee-popover {
+  & &__edit-link {
```

Comment:
I think there were some changes that made my previous comment outdated, so just adding this here again. If the styles aren't getting applied without the additional qualifier `.advanced-gas-fee-popover` then that would indicate some problem with specificity. Do you know what other styles are overriding this? If we could correct it at that spot then we shouldn't have to override the specificity here.